src/Microsoft.OpenApi/Models/OpenApiRequestBody.cs

```diff
@@ -123,23 +124,37 @@ public IEnumerable<IOpenApiParameter> ConvertToFormDataParameters(IOpenApiWriter
 
             foreach (var property in Content.First().Value.Schema.Properties)
             {
-                var paramSchema = new OpenApiSchema(property.Value);
+                var paramSchema = property.Value.CreateShallowCopy();
                 if ((paramSchema.Type & JsonSchemaType.String) == JsonSchemaType.String
                     && ("binary".Equals(paramSchema.Format, StringComparison.OrdinalIgnoreCase)
                     || "base64".Equals(paramSchema.Format, StringComparison.OrdinalIgnoreCase)))
                 {
-                    paramSchema.Type = "file".ToJsonSchemaType();
-                    paramSchema.Format = null;
+                    var updatedSchema = paramSchema switch {
+                        OpenApiSchema s => s, // we already have a copy
+                        // we have a copy of a reference but don't want to mutate the source schema
+                        // TODO might need recursive resolution of references here
```

Comment:
on this one I have a question pending internally with @darrelmiller

Reply:
confirmed with Darrel, references of references are possible.
I'll stand up another PR to update the return type of the target field, and enable some recursion work.

Reply:
this is actually what's being discussed in #2062 !!